### PR TITLE
I18n start

### PR DESCRIPTION
**Pull request by kindly**
Start i18n interface translation.

Add language dropdown.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,36 @@ Flask in development mode
 FLASK_ENV=development FLASK_APP=extension_explorer/views.py flask run
 ```
 
+# I18n
+
+## Compile messages:
+
+This needs to be done on each computer for i18n to work. If updating messages from transifex this has to be done last.
```
Comment (odscjames) [inline README.md]: Can this be clarified?

> This needs to be done on each computer for i18n to work

Does This mean the "compile messages" section or the whole "I18N" section?

> If updating messages from transifex this has to be done last.

Done lost compared to what? 


Is this whole I18N section basically optional? If your a dev just running the site you don't need to touch it, but 
  *  if you want to get updated translations 
  *  you change the messages that are in the templates

... then you need to run the whole section?

So it's either - don't change the messages and just totally ignore the I18N section - or - do change them and run the whole I18N section? Is that right? Can that be clarified if so?

File: README.md [hunk 2]
```diff
@@ -27,6 +27,36 @@ Flask in development mode
...
+
+```
+cd extension_explorer
+pybabel compile -d translations
+```
+
+## Extract messages and push to transifex
```
Comment (odscjames) [inline README.md]: Wouldn't you need some kind of key set up to push to Transifex?

**Comment by kindly** [comment]
I think for this case the SSOT makes sense to be this repo.  It is like this for most web projects, including cove and the standard, as it makes sense for the translations to be alongside the source.  It also means you do not need transifex API access to contribute.

For the collector we have made transifex the SSOT, but that is an exception, as the data we are gathering is external and not in the repo itself.

File: extension_explorer/templates/extension_codelists.html [hunk 3]
```diff
@@ -16,17 +16,17 @@ <h2 id="{{codelist}}"> {{codelist}} </h2>
...
+              {% endfor %}
             </tr>
         </thead>
         <tbody>
           {% for key, value in extension_version.codelists[codelist]["items"].items() %}
             <tr>
-                <th scope="row">{{key}}</th>
-                <td>{{value[lang].Title}}{{value[lang].Title_en}}</td>
-                <td>{{value[lang].Description}}{{value[lang].Description_en}}</td>
+              {% for fieldname, fieldname_value in extension_version.codelists[codelist]["fieldnames"].items() %}
```
Comment (jpmckinney) [inline extension_explorer/templates/extension_codelists.html]: This loses the `<th scope="row">`, but those semantics aren't needed.

**Pull request merged**